### ISSUE TITLE
Consider whether DefaultResolutionSet should expose a getResolution(Boolean, Boolean) public API

### ISSUE THREAD

**Issue by QuintinWillison** [ably/ably-asset-tracking-android#101]
spawned by: https://github.com/ably/ably-asset-tracking-android/pull/97#discussion_r544024410

Deferred to this issue so we can unblock that PR.

┆Issue is synchronized with this [Jira Task](https://ably.atlassian.net/browse/ATS-149) by [Unito](https://www.unito.io/learn-more)
